model/src/main/kotlin/Package.kt

```diff
@@ -75,8 +75,9 @@ data class Package(
     val declaredLicensesProcessed: ProcessedDeclaredLicense = DeclaredLicenseProcessor.process(declaredLicenses),
 
     /**
-     * The concluded license as an [SpdxExpression]. It can be used to correct the license of a package in case the
-     * [declaredLicenses] found in the packages metadata or the licenses detected by a scanner do not match reality.
+     * The concluded license as an [SpdxExpression]. It can be used to override the [declared licenses]
+     * [Package.declaredLicenses] and / or the [detected license expression][LicenseFinding.license] of a package in
+     * case either are incorrect.
```

Comment:
Maybe "in case either are incorrect." can be dropped, because overriding a correct license with identity or something incorrect obviously does not make any sense.

Reply:
Agreed, esp. as I was a bit unhappy with the term "incorrect" (I simply had "wrong" locally before) as it left unclear in what regard the license is "incorrect" (completely wrong, or just missing additional licenses). So it's probably better to drop that bit completely.